examples/README.md

```diff
@@ -178,6 +178,12 @@ of samples per parameter with the `samples` value and optionally the random seed
   - `loguniform`:  Uniformly samples in log space between `min` and `max` as specified in the parameter dict.
   - `randint`: Randomly samples integers between `min` (included) and `max` (excluded).
 
+### Named Configurations
+`sacred`, the on which experiments are based on, allows to define subgroups of configurations via its [named configurations](https://sacred.readthedocs.io/en/stable/configuration.html#named-configurations) feature. These can either be defined in external files (yaml, json, ...) or in functions decorated with `experiment.named_config`. SEML also supports this functionality by defining parameter groups that have the prefix `'$named_config'`. Two config values can be defined for such parameter groups:
```

Comment:
`sacred`, ~the~ on which the experiments ...

We should mention that one can add postfix to `$named_config[_<postfix>]` to allow for multiple configs.

Do we support the shorthand `$named_config: cora.yaml`, or does one always have to use `$named_config.name: cora.yaml`?

Reminder: change `$named_config` here.

Reply:
- `$named_config` as literal: I am not a huge fan of this, as it will make queries for the MongoDB (both with a client and in seml) more contrived and prone to errors. Regarding omegaconf: You are techincally right, but here it only should affect the values, not the keys / paths. I do not see the advantage of using a symbol that is reserved by MongoDB as a useful prefix. If anything, it would be very inconsistent und potentially confusing, no? I am also not super sold on `%`. Can't it be something readable, maybe something as simple as `NAMED_CONFIG`. The user can anyway change it in `settings.py` if they desire so. 

- Resolution order: The procedure in which we resolve directly matches sacred, which I'd argue is the most consistent way: https://sacred.readthedocs.io/en/stable/command_line.html#combination-with-regular-updates

- Shorthand: I guess we can do that. One issue I see is that you can anyway not use `$named_config_dataset: cora_ml` and `$named_config_dataset.priority: 3` in the same configuration, as `seml`'s yaml parser will complain about this. Are we fine with not supporting that, or put differently, is the convenience gain that large by omitting the `.name` to introduce some small inconsistency?